draft-ietf-httpbis-digest-headers.md

```diff
@@ -141,20 +141,10 @@ HTTP/1.1 ([RFC7231], Appendix B) obsoleted it:
 [RFC3230] provided a more flexible solution introducing the concept of
 "instance", and the fields `Digest` and `Want-Digest`.
 
-## This Proposal
+## This Document
```

Comment:
This is still a weird section header :smile: 

 https://github.com/httpwg/http-extensions/pull/1477 tackles the problem differently